cip/2.testable/CIP2015-06-24-call-procedures.adoc

```diff
@@ -102,65 +128,40 @@ This calls `myProc` for each record produced by the preceding `MATCH` clause wit
 Each call may yield multiple procedure result records with a single result field `last`.
 For each of them, a new statement result record is created that contains the original variables already in scope (i.e. `n`, `r`, `g`) as well as the projected result field `last` renamed as `lastLogin`.
 
+Omitting the `YIELD` subclause means that no new variables are introduced into the scope.
+The procedure call still has the same effect on the cardinality of the query execution in this case.
+
 The `YIELD` subclause is *always* omitted if the procedure returns only records with no result fields (i.e. is of type `()`).
 
 ==== Standalone calls
 
 Procedures may also be called standalone, i.e. without taking arguments from or combining their results with other parts of a larger query.
-In this case, the trailing `RETURN` clause may be omitted and all variable in scope are returned implicitly by the statement.
-Furthermore, we propose two optional simplifications that are available for standalone calls only.
-These simplifications are outlined in the following:
+In this case, the trailing `RETURN` clause is omitted and all fields projected in the `YIELD` subclause of the `CALL` statement are implicitly returned by the query.
+A further simplification allowed in the standalone form is to use `YIELD *` to denote that all fields returned by the procedure are to be returned by the query.
+The `YIELD *` form is _only_ allowed in the standalone form of `CALL`.
 
-===== Calling with implicitly passed arguments (parameters)
-
-Standalone calls may omit passing arguments explicitly.  In this case, all required procedure arguments are taken implicitly from statement parameters with the same name.
+If the `YIELD` subclause is omitted in the standalone form of `CALL`, the semantics are the same as in the _In-query_ form, i.e. the query will not project any fields into the result, but the query will still return as many (empty) rows as produced by the called procedure.
```

Comment:
I don't think "in-query" needs to be capitalised here ('standalone' hasn't been)